src/client/Client.cc

```diff
@@ -6415,9 +6422,14 @@ void Client::flush_cap_releases()
       } else {
         session.con->send_message2(std::move(session.release));
       }
+      nr_caps += session.release->caps.size();
```

Comment:
`session.release` may be null due to the previous `else` block. It causes this assertion:

````
2020-11-14T02:27:59.720 INFO:teuthology.orchestra.run.smithi164.stderr:ceph_test_trim_caps: /build/ceph-16.0.0-7160-g4090b2dd/obj-x86_64-linux-gnu/boost/include/boost/smart_ptr/intrusive_ptr.hpp:199: T* boost::intrusive_ptr<T>::operator->() const [with T = MClientCapRelease]: Assertion `px != 0' failed.
````

From: /ceph/teuthology-archive/pdonnell-2020-11-14_02:06:06-fs-wip-pdonnell-testing-20201113.235156-distro-basic-smithi/5620640/teuthology.log

Reply:
FIxed, thanks.